ksqldb-common/src/main/java/io/confluent/ksql/parser/VariableParser.java

```diff
@@ -0,0 +1,30 @@
+package io.confluent.ksql.parser;
```

Comment:
Missing header. Might want to run checkstyle locally to expedite getting a passing build (once master is unblocked) since we're trying to get this in by tomorrow. (I see a missing space after `catch` in the newly added try-catch block in the file below.)